src/app/components/user/cognito-user-invite.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from 'react';
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  MenuItem,
+  ThemeProvider,
+  makeStyles,
+} from '@material-ui/core';
+import PersonAddIcon from '@material-ui/icons/PersonAdd';
+import EmailIcon from '@material-ui/icons/Email';
+import { arrayOf, func, string } from 'prop-types';
+import { useFormik } from 'formik';
+import * as yup from 'yup';
+
+import theme from '../../themes/theme';
+import { ChplDialogTitle, ChplTooltip, ChplTextField } from '../util';
+
+const useStyles = makeStyles(() => ({
```

Comment:
I don't know why this is the format that we started with, but it's unnecessary. This has less code and still works:

```suggestion
const useStyles = makeStyles({
```

I think one of the parentheses on line 28 will need to be removed too